docs/source/scalarstats.rst

```diff
@@ -158,6 +158,9 @@ Quantile and Friends
     w = rand(n)
     xk = median(x, weights(w))
 
+.. function:: wquantile(x, p, w)
+
+  Compute the weighted quantile of ``x``, using weights given by a weight vector ``w`` (of type ``WeightVec``).  Weights must not be negative. The weights and data vectors must have the same length. The quantile for :math:`p` is defined as follows. Normalizing ``w`` so that the sum of weights equals :math:`n`, the length of ``x``, and denoting :math:`x(h)` the smallest element of ``x`` such that the cumulated sum of weights up to :math:`x(h)` is strictly superior to :math:`n p + m`, the function returns :math:`(1-\gamma) x(h-1) + \gamma x(h)`, with  :math:`m=1-p` and :math:`\gamma = n p + m - \sum_{i < h} w_i`. This corresponds to the unweighted quantile function when ``w`` is 1 (type 7 in R).
```

Comment:
You still don't use the plural, and don't specify that a vector of probabilities can be passed too.

Also, regarding the type, you could give references to other software than R. Wikipedia says this method is "R-7, Excel, SciPy-(1,1), Maple-6". https://en.wikipedia.org/wiki/Quantile